node_modules/oae-activity/emailTemplates/mail.html.jst

```diff
@@ -8,21 +8,21 @@
         greeting = shared.ensureAbsoluteLinks(greeting, baseUrl);
 
         var str = '';
-        str += '<tr id="header" class="row">';
-        str += '    <td align="center">';
-        str += '        <table cellpadding="0" cellspacing="0">';
-        str += '            <tr id="logo">';
-        str += '                <td><img src="' + institutionalLogoURL + '" /></td>';
-        str += '            </tr>';
-        str += '            <tr id="greeting">';
-        str += '                <td><h1>' + greeting + '</h1></td>';
-        str += '            </tr>';
-        str += '            <tr id="summary" class="muted">';
-        str += '                <td class="wrapped">' + shared.getEmailSummary(util, user.emailPreference, activities, baseUrl) + '</td>';
-        str += '            </tr>';
-        str += '        </table>';
-        str += '    </td>';
-        str += '</tr>';
+        str += '<tr id="header" class="row">\n';
```

Comment:
Rather than manually appending `\n` everywhere, you could just push the lines in an array and join them at the end